Session.js

```diff
@@ -0,0 +1,35 @@
+class Session { 
+// To be utilized when unwrapping JSON from the future servlets. Will be on the client side and be used as a buffer  
+// for client to server information needed for other components of Virtual Movie Night.
+ 
+    constructor() {
+        const expected = {
+            sessionId: 'JTC6',
+            screenNameOfController: 'chris',
+            ipOfVM: '01',
+            listOfAttendees: ['chris', 'bryan']
+            };
```

Comment:
Do you have this because there's no servlet yet? This is a bit suspect.
 
Add setters for each of the fields then add a static function called fromObject(obj) then construct and return a Session object there.